packages/suite/src/components/wallet/Fees/StandardFee/EthereumFeeCards.tsx

```diff
@@ -14,122 +14,109 @@ import { FeeOptionType, getFeeLevelTranslationId } from '../Fees';
+            setCachedGasLimit(transactionInfo.feeLimit);
+        }
+        if (!isDirty) {
+            setCachedGasLimit(undefined);
         }
+    }, [transactionInfo, isDirty]);
 
-        const getTimeEstimate = (fee: FeeOptionType) => {
-            if (fee.maxWaitTimeEstimate) {
-                return `~${formatDurationStrict(fee.maxWaitTimeEstimate / 1000, locale)}`;
-            }
+    if (!feeOptions.length) {
+        return null;
+    }
+
+    const getTimeEstimate = (fee: FeeOptionType) => {
+        if (fee.maxWaitTimeEstimate) {
+            return `~${formatDurationStrict(fee.maxWaitTimeEstimate / 1000, locale)}`;
+        }
 
-            return undefined;
-        };
+        return undefined;
+    };
 
-        return (
-            <>
-                <FeeCardsWrapper
-                    $columns={columns ?? 1}
-                    ref={ref}
-                    data-testid="@wallet/fee-details"
-                >
-                    {feeOptions.map(fee => (
-                        <FeeCard
-                            value={fee.value}
-                            key={fee.value}
-                            isSelected={selectedLevel.label === fee.value}
-                            changeFeeLevel={changeFeeLevel}
-                            topLeftChild={
-                                <span data-testid={`@fee-card/${fee.value}`}>
-                                    <Translation id={getFeeLevelTranslationId(fee.value)} />
-                                </span>
-                            }
-                            topRightChild={getTimeEstimate(fee)}
-                            bottomLeftChild={
-                                <FiatValue
-                                    disableHiddenPlaceholder
-                                    amount={fee.networkAmount || ''}
-                                    symbol={symbol}
-                                    showApproximationIndicator
-                                />
-                            }
-                            bottomRightChild={
-                                <FeeRate
-                                    feeRate={isEip1559(fee) ? fee.maxFeePerGas : fee.feePerUnit}
-                                    networkType={networkType}
-                                    symbol={symbol}
-                                />
-                            }
-                            tooltipContent={
-                                isDebug && isEip1559(fee) ? (
-                                    <Grid columns={2} gap={spacings.xs}>
-                                        <Row>
-                                            <Badge variant="warning" size="small">
-                                                <Translation id="TR_DEBUG_ONLY" />
-                                            </Badge>
-                                        </Row>
-                                        <span />
```

Comment:
Removing this caused this

<img width="325" alt="Screenshot 2025-04-05 at 11 04 14" src="https://github.com/user-attachments/assets/5ce1bfa1-10a1-4a37-9b25-98508c78c7f6" />